bd_spot_wrapper/spot_wrapper/spot.py

```diff
@@ -836,6 +922,19 @@ def get_camera_intrinsics(self, source, quality=None, pixel_format=None):
         cam_intrinsics = image_response.source.pinhole.intrinsics
         return cam_intrinsics
 
+    def get_camera_intrinsics_as_3x3(self, camera_intrinsics) -> np.ndarray:
```

Comment:
We could combine this with the one above by adding a ‘as_matrix’ flag to the function call. Return response when that flag is false otherwise return a 3x3 matrix.

Reply:
added